# Fixed a possible invalid lock sequence if the main menu is open and the user
# switches to a channel that is currently not available, using the Channel+/- keys.
#
--- device.c	2018/07/16 09:29:57	5.1
+++ device.c	2019/05/06 13:10:36
@@ -831,7 +831,7 @@
            result = true;
         }
      else if (n != first)
-        Skins.Message(mtError, tr("Channel not available!"));
+        Skins.QueueMessage(mtError, tr("Channel not available!"));
      }
   return result;
 }
